app/javascript/dashboard/store/modules/conversations/index.js

```diff
@@ -15,6 +15,7 @@ const state = {
   currentInbox: null,
   selectedChatId: null,
   appliedFilters: [],
+  contextMenuChatId: null,
```

Comment:
I use Vuex to show the snooze option dynamically on the CMD bar. Only show the snooze option in the CMD bar when we click the snooze button from the context menu (NB: home route)